### PR TITLE
fix(InvitationBubbleView): inconsistency in message link previews buttons

### LINKED ISSUE #9157: Inconsistency in message link previews buttons
# Bug Report

Comparing two components when bottom button hovered:

1. Community Invitation: *All corners rounded*
image

2. Enable automatic image unfurling: *Only bottom corners rounded*
image

### PR DESCRIPTION
**Pull request by caybro**
make the button rounded only at the bottom

Fixes #9157

### What does the PR do

Fixes rounded corners in InvitationBubbleView (Go to Community...)

### Affected areas

InvitationBubbleView

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

image

### CODE REVIEW

**Review by borismelnik — APPROVED** [review]
👍

File: ui/imports/shared/views/chat/InvitationBubbleView.qml [hunk 1]
```diff
@@ -176,21 +176,35 @@ Control {
...
             Layout.fillWidth: true
             Layout.preferredHeight: 44
+            clip: true
+
+            StatusFlatButton {
+                id: joinBtn
+                width: parent.width
+                height: (parent.height+d.radius)
+                anchors.top: parent.top
+                anchors.topMargin: -d.radius
```
Comment (igor-sirotin) [inline ui/imports/shared/views/chat/InvitationBubbleView.qml:189]: tricky!

**Review by igor-sirotin — APPROVED** [review]

File: ui/imports/shared/views/chat/InvitationBubbleView.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/views/chat/InvitationBubbleView.qml:189]: Indeed but kinda standard in QML 😊

Comment (caybro) [reply ui/imports/shared/views/chat/InvitationBubbleView.qml:189]: The other way would be to use some OpacityMask but this involves a GL shader, while this one has `clip`ping. Actually dunno which one would be better/faster but hey, it works :)

**Pull request merged**